scripts/compile-references.js

```diff
@@ -69,7 +62,7 @@ compileTypeScriptReferences().catch(error => {
  * This script main entry point.
  */
 async function compileTypeScriptReferences() {
-    await Promise.all([THEIA_MONOREPO, ...Object.values(YARN_WORKSPACES)].map(async package => {
+    await Promise.all(Object.values(YARN_WORKSPACES).map(async package => {
```

Comment:
`YARN_WORKSPACES` still requires to use `yarn --silent workspaces info`, which is an implicit yarn dependency.